Makefile

```diff
@@ -31,3 +31,21 @@ tag:
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
 	docker build -t ${IMG} .
+
+sbom-tool:
+	curl -Lo sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
+	chmod +x sbom-tool
+
+.PHONY: sbom
+sbom: sbom-tool
+	./sbom-tool generate \
```

Comment:
I wonder if we could get the output files to be neatly put in `./target/release/` or similar.. but doesn't really matter. Also, `sbom-tool` help falls a bit short...